### PR TITLE
Fix codeclimate report

### PR DESCRIPTION
**Pull request by artofhuman**
https://github.com/codeclimate/ruby-test-reporter/blob/master/CHANGELOG.md#v100-2016-11-03

It should fix test in master

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 if ENV["CODECLIMATE_REPO_TOKEN"]
-  require "codeclimate-test-reporter"
-  CodeClimate::TestReporter.start
```
Comment (laserlemon) [inline spec/spec_helper.rb]: Does this no longer report to Code Climate then?

**Comment by laserlemon** [comment]
I'm hesitant to just remove Code Climate. I'll review the breakage as part of v4 development.

**Comment by michaelherold** [comment]
This is just missing a step. You'll want to add an:
```yaml
after_script:
  - bundle exec codeclimate-test-reporter
```
directive to your `.travis.yml` configuration.

**Comment by laserlemon** [comment]
@michaelherold Could you please add that to your PR so we can see if it runs as expected? Thank you!

**Comment by michaelherold** [comment]
@artofhuman That request should actually be to you. 👍

**Comment by almostwhitehat** [comment]
`after_script` directives execute even if the `script` fails. We probably don't want to push CC coverage on a failed rspec run. I'd just chain the two of them together:

```
script:
  - bundle exec rspec
  - bundle exec codeclimate-test-reporter
```

**Comment by laserlemon** [comment]
There's also `after_success`, which seems more appropriate.
https://docs.travis-ci.com/user/customizing-the-build

**Comment by laserlemon** [comment]
In fact, from the [docs](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step):

> You can specify multiple script commands as well:
> 
> ```yaml
> script:
> - bundle exec rake build
> - bundle exec rake builddoc
> ```
> 
> When one of the build commands returns a non-zero exit code, the Travis CI build runs the subsequent commands as well, and accumulates the build result.
> 
> In the example above, if `bundle exec rake build` returns an exit code of 1, the following command `bundle exec rake builddoc` is still run, but the build will result in a failure.

**Comment by michaelherold** [comment]
I'm glad I commented on this because I'm learning about features of Travis that I didn't know about.

Thanks!

**Comment by laserlemon** [comment]
Merged #132.

**Pull request closed without merging**